kfs-purap/src/main/resources/edu/arizona/kfs/module/purap/document/validation/configuration/PurapValidatorOverrides.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file PurapValidatorOverrides.xml does not exist in KFS 6. This file was created in our KFS 3 implementation to hold all the UA PURAP validator overrides. This may have sounded like a good idea a first, but quickly got out of hand as more and more overrides were added. The overrides should go in edu.arizona validator files that match the names of the original validation files. Please see below.